mailer.rst

```diff
@@ -1266,6 +1266,46 @@ The following transports only support tags:
 
 * OhMySMTP
 
+Draft Emails
+------------
+
+.. versionadded:: 6.1
+
+    ``Symfony\Component\Mime\DraftEmail`` was introduced in ``symfony/mime`` 6.1.
```

Comment:
perhaps remove the ``symfony/mime``, I've never seen such package desc inside versionadded code block before